incuna-sass/_normalize.sass

```diff
@@ -44,7 +44,7 @@ html
 // Links
 
 // Remove the gray background color from active links in IE 10.
-@if support-legacy-browser("ie", "10")
+.lt-ie11 &
```

Comment:
I'm not sure that our templates will add this class, as I think ie9 and 10 don't detect like this? we might need to have a look at the base template